pandas/core/base.py

```diff
@@ -316,13 +331,14 @@ def _try_aggregate_string_function(self, arg, *args, **kwargs):
 
         raise ValueError("{arg} is an unknown string function".format(arg=arg))
 
-    def _aggregate(self, arg, *args, **kwargs):
+    def _aggregate(self, arg, axis=0, *args, **kwargs):
```

Comment:
I still think this should be a separate PR. I know you mentioned there was some interweaving of test dependency, but I feel like we are injecting this keyword in here without any regard to existing test coverage for axis=1.

Reply:
I can look into making separate PR. That will have to be pulled in before this one, so the tests of this PR won't break.

Reply:
If timing is a concern you can also xfail the axis=1 tests. Rebase thereafter would be minor

Reply:
This has now been created as #21222